main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"syscall"
+)
+
+const ProcessesLimit = "10"
+
+func main() {
+    if len(os.Args) < 2 {
+        panic("Few arguments")
+    }
+
+    switch os.Args[1] {
+    case "run":
+        Run(os.Args[2], os.Args[3:])
+    case "child":
+        Child(os.Args[2], os.Args[3:])
+    default:
+        panic("Bad usage")
+    }
```

Comment:
`panic` is not the right way to exit an app with an error. A panic donates a "programming" error. An error that the developer of the app didn't account for. Hence, any decent app should never panic.

The right way to do it is as follows:
```go
func main() {
    if err := app(); err != nil {
         // notice i write the error explicitly to the stderr of the app 
         fmt.Fprintf(os.Stderr, err.Error())
         os.Exit(1) // I exit with a non zero error exit code. this is interpreted by a terminal as
                         // unsuccessful exit of the app. hence your cli can be used in bash scripts 
    }
}
```